code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -88,7 +125,10 @@
 			world << pick(creepyasssounds)
 		else
 			lastsinPerson = world.time
-			var/mob/living/carbon/human/sinPerson = pick(player_list)
+			var/mob/living/carbon/human/sinPerson = pick(living_mob_list)
+			if(!sinPerson.ckey)
+				while(!sinPerson.ckey)
+					sinPerson = pick(living_mob_list)

```

Comment:
It should theoretically never cause an overflow; If there's a person in the game, it will find them and I've experienced no issues with it.

It would overflow if there was only a single admin ghost in the game and no other active players. I'll add a redundancy check just in case though.